docs/resources.html

```diff
@@ -37,59 +40,62 @@ <h2 id="general">General</h2>
 				<h3 id="english-language-editing">English language editing</h3>
 				<p>If your journal publishes in English, you can consider offering an additional free editing service for manuscripts submitted by those for whom English is a second language. For example, see the <a href="https://onlinelibrary.wiley.com/page/journal/14756811/homepage/forauthors.html" target="_blank">International Section guidelines</a> at <a href="https://onlinelibrary.wiley.com/journal/14756811" target="_blank">Personal Relationships</a> and this article about <a href="https://view.publitas.com/iarr/rrn-printernationalscholarship-november2021/page/1" target="_blank">how the International Section is contributing to elevating international scholarship</a>.</p>
+				<h3 id="retroactive-name-changes">Retroactive name changes</h3>
+				<p>Although retroactive name changes may seem strangely placed in a section on diversity, the historical inability to retroactively change names in publications is likely to disproportionately impact marginalized individuals. Women are historically more likely than men to change their name after marriage and/or divorce. Although many authors who have changed their names after marriage or divorce have no problem authoring publications under different names, for others viewing their previous name will be traumatic (e.g., where a marriage ended because of domestic violence). In addition, transgender individuals often choose to change their names, and being referred to by their previous name can also be traumatic for them.</p>
+				<p>This trauma can be avoided by implementing a name change policy. These policies can be implemented at the publisher level (e.g. see these from <a href="https://www.wiley.com/network/societyleaders/publishing-strategy/new-author-name-change-policy-supports-a-more-inclusive-publishing-environment" target="_blank" title="Wiley name change policy">Wiley</a> and <a href="https://www.elsevier.com/about/policies/inclusive-author-name-changes" target="_blank" title="Elsevier name change policy">Elsevier</a>), the society level (e.g. see these from the <a href="https://www.apa.org/pubs/journals/resources/policy-author-name-changes" target="_blank" title="APA name change policy">American Psychological Association</a> and the <a href="https://axial.acs.org/2021/06/21/acs-publications-name-change-policy-advances-inclusion-in-scholarly-publishing/" target="_blank" title="ACS name change policy">American Chemical Society</a>), or the individual journal level.</p>
```

Comment:
```suggestion
				<p>This trauma can be avoided by implementing a name change policy. These policies can be implemented at the publisher level (e.g., see these from <a href="https://www.wiley.com/network/societyleaders/publishing-strategy/new-author-name-change-policy-supports-a-more-inclusive-publishing-environment" target="_blank" title="Wiley name change policy">Wiley</a> and <a href="https://www.elsevier.com/about/policies/inclusive-author-name-changes" target="_blank" title="Elsevier name change policy">Elsevier</a>), the society level (e.g., see these from the <a href="https://www.apa.org/pubs/journals/resources/policy-author-name-changes" target="_blank" title="APA name change policy">American Psychological Association</a> and the <a href="https://axial.acs.org/2021/06/21/acs-publications-name-change-policy-advances-inclusion-in-scholarly-publishing/" target="_blank" title="ACS name change policy">American Chemical Society</a>), or the individual journal level.</p>
```

Commas after e.g. for consistency